src/plugins/python/weechat.pyi

```diff
@@ -48,202 +48,454 @@ WEECHAT_HOOK_SIGNAL_POINTER: str
 
 
 def register(name: str, author: str, version: str, license: str, description: str, shutdown_function: str, charset: str) -> int:
-    """`register in WeeChat plugin API reference <https://weechat.org/doc/api/#_register>`_"""
+    """`register in WeeChat plugin API reference <https://weechat.org/doc/api/#_register>`_
+    ::
+        ----
+
+        [NOTE]
```

Comment:
The parsing of example seems to have failed for this `register` function.

Reply:
Ah, sorry I missed that. It's fixed now.